maven/bnd-maven-plugin/src/main/java/aQute/bnd/maven/plugin/AbstractBndMavenPlugin.java

```diff
@@ -565,6 +565,15 @@ private static StringBuilder addHeaderAttribute(StringBuilder builder, String ke
 
 	private void attachArtifactToProject(Jar bndJar) {
 		File artifactFile = createArtifactFile();
+		File outputDir = artifactFile.getParentFile();
+
+		if (!outputDir.exists()) {
+			try {
+				IO.mkdirs(outputDir);
+			} catch (IOException ioe) {
+				Exceptions.duck(ioe);
```

Comment:
```suggestion
				throw Exceptions.duck(ioe);
```

We always want to use the construct "throw the duck" as this tells the compiler there is a flow control change.

Reply:
good catch, I missed that one. fixed!